src/workloads/query/multiplanner/Simple.yml

```diff
@@ -0,0 +1,400 @@
+          x41: *distribution
+          x42: *distribution
+          x43: *distribution
+          x44: *distribution
+          x45: *distribution
+          x46: *distribution
+          x47: *distribution
+          x48: *distribution
+          x49: *distribution
+          x50: *distribution
+          x51: *distribution
+          x52: *distribution
+          x53: *distribution
+          x54: *distribution
+          x55: *distribution
+          x56: *distribution
+          x57: *distribution
+          x58: *distribution
+          x59: *distribution
+          x60: *distribution
+          x61: *distribution
+          x62: *distribution
+          x63: *distribution
+        Indexes:
+          - keys: {x1: 1}
+          - keys: {x2: 1}
+          - keys: {x3: 1}
+          - keys: {x4: 1}
+          - keys: {x5: 1}
+          - keys: {x6: 1}
+          - keys: {x7: 1}
+          - keys: {x8: 1}
+          - keys: {x9: 1}
+          - keys: {x10: 1}
+          - keys: {x11: 1}
+          - keys: {x12: 1}
+          - keys: {x13: 1}
+          - keys: {x14: 1}
+          - keys: {x15: 1}
+          - keys: {x16: 1}
+          - keys: {x17: 1}
+          - keys: {x18: 1}
+          - keys: {x19: 1}
+          - keys: {x20: 1}
+          - keys: {x21: 1}
+          - keys: {x22: 1}
+          - keys: {x23: 1}
+          - keys: {x24: 1}
+          - keys: {x25: 1}
+          - keys: {x26: 1}
+          - keys: {x27: 1}
+          - keys: {x28: 1}
+          - keys: {x29: 1}
+          - keys: {x30: 1}
+          - keys: {x31: 1}
+          - keys: {x32: 1}
+          - keys: {x33: 1}
+          - keys: {x34: 1}
+          - keys: {x35: 1}
+          - keys: {x36: 1}
+          - keys: {x37: 1}
+          - keys: {x38: 1}
+          - keys: {x39: 1}
+          - keys: {x40: 1}
+          - keys: {x41: 1}
+          - keys: {x42: 1}
+          - keys: {x43: 1}
+          - keys: {x44: 1}
+          - keys: {x45: 1}
+          - keys: {x46: 1}
+          - keys: {x47: 1}
+          - keys: {x48: 1}
+          - keys: {x49: 1}
+          - keys: {x50: 1}
+          - keys: {x51: 1}
+          - keys: {x52: 1}
+          - keys: {x53: 1}
+          - keys: {x54: 1}
+          - keys: {x55: 1}
+          - keys: {x56: 1}
+          - keys: {x57: 1}
+          - keys: {x58: 1}
+          - keys: {x59: 1}
+          - keys: {x60: 1}
+          - keys: {x61: 1}
+          - keys: {x62: 1}
+          - keys: {x63: 1}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: Queries
+  Type: CrudActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3, 5, 7]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        # TODO more repeats
+        Repeat: 2
+        Database: *db
+        Collection: *coll
+        Operations:
+        - OperationMetricsName: RunQueries
```

Comment:
You also don't need to specify OperationMetricsName. It is an optional field, so if it is not something meaningful, let's just delete it and it will default to OperationName